src/commands/data/connectors/index.ts

```diff
@@ -96,13 +98,17 @@ export default class ConnectorsList extends BaseCommand {
           header: 'Postgres Add-On',
           get: row => row.postgres_addon.name,
         },
+        tables: {header: 'Tables'},
+        excluded_columns: {header: 'Excluded Columns'},
       })
     } else {
-      connectorInfo.forEach(v => {
+      connectorInfo.forEach(c => {
         cli.styledObject({
-          'Connector Name': v.name,
-          'Kafka Add-On': v.kafka_addon.name,
-          'Postgres Add-On': v.postgres_addon.name,
+          'Connector Name': c.name,
+          'Kafka Add-On': c.kafka_addon.name,
+          'Postgres Add-On': c.postgres_addon.name,
+          Tables: c.tables,
```

Comment:
what happens if there are 20 tables? Do we care?

Reply:
Good question. They will be presented, space delimited, here. I imagine 20 tables might look a little gnarly but it should wrap with the width of the terminal.

The purpose of this improvement is to give the caller information about all their connectors and note the differences when the same add-ons are used (eg: the Tables list will be different).

Reply:
I figured that was the answer, and why I said "do we care". Sounds like listing the data is really what we care about and not really how pretty it looks.